guides/common/assembly_administering-hosts.adoc

```diff
@@ -17,8 +17,6 @@ include::modules/proc_changing-the-module-stream-for-a-host.adoc[leveloffset=+1]
 include::modules/proc_enabling-custom-repositories-on-content-hosts.adoc[leveloffset=+1]
 
 ifdef::katello,satellite,orcharhino[]
```

Comment:
@maximiliankolb Do you know why the procedure was included within an ifdef/endif here? That's the only concern I have a this point -- you're not using a conditional in guides/common/assembly_configuring-host-collections.adoc and I don't see it in doc-Managing_Hosts/master.adoc either. I don't think there is a reason to use it but I still want to check that it's okay.

Reply:
Yes, looks like it's indeed a Katello feature.